languages/de.yaml

```diff
@@ -15,6 +15,13 @@ PLUGIN_GANALYTICS:
   POSITION_HELP: "Position des Codes im HTML-Dokument (Head oder Body)"
   POSITION_HEAD: "Kopf"
   POSITION_BODY: "Inhaltsende"
+  DO_NOT_TRACK_SECTION: "Do Not Track"
+  DO_NOT_TRACK: "Aktiviere 'Do Not Track'"
+  DO_NOT_TRACK_HELP: "Erlaubt Klienten, durch das senden des 'Do Not Track'-Headers, vom Tracking ausgeschlossen zu werden."
+  DO_NOT_TRACK_LOG: "Aktiviere Protokollierungsmodus"
+  DO_NOT_TRACK_LOG_HELP: "Beachten Sie, dass, während Sie das Plugin im Protokollierungsmodus verwenden, Daten an Google Analytics gesendet werden, auch wenn 'Do Not Track' aktiviert ist. Erst wenn Sie den Protokollierungsmodus deaktivieren, werden keine Daten an Google Analytics gesendet. Sie sollten nur den Protokollierungsmodus verwenden, um herauszufinden, welcher Anteil von Besuchen / Benutzern Sie verlieren, nachdem Sie die Protokollierung deaktiviert und die 'Do Not Track'-Unterstützung aktiviert haben."
+  DO_NOT_TRACK_DIMENSION: "Google Analytics Dimension"
+  DO_NOT_TRACK_DIMENSION_HELP: "Google Analytics Dimension to log the 'Do Not Track'-Status (https://goo.gl/i15S6B)"
```

Comment:
Please use this translation `Google Analytics Dimension um den 'Do Not Track'-Status zu protokollieren (https://goo.gl/i15S6B)`